templates/ocs-deployment/filesystem.yaml

```diff
@@ -0,0 +1,54 @@
+#################################################################################################################
+# Create a filesystem with settings with replication enabled for a production environment.
+# A minimum of 3 OSDs on different nodes are required in this example.
+#  kubectl create -f filesystem.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: myfs
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+spec:
+  # The metadata pool spec. Must use replication.
+  metadataPool:
+    replicated:
+      size: 3
+  # The list of data pool specs. Can use replication or erasure coding.
+  dataPools:
+    - failureDomain: host
+      replicated:
+        size: 3
```

Comment:
change this as a param "replicasize"

Reply:
yes Ram, I have made func create_fs() with all  configurable params needed

Reply:
+1 on parameterized here as well